invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/components/DocumentAuthors.js

```diff
@@ -4,16 +4,20 @@ import PropTypes from 'prop-types';
 
 export class DocumentAuthors extends Component {
```

Comment:
Maybe we can move the `DocumentAuthors` to the common components of Front and Back sites, so we can use them everywhere.